### DISCUSSION TITLE
Why is the data I get back from a gridpoints endpoint call different that what I see for the same location at https://forecast.weather.gov/MapClick.php

### DISCUSSION THREAD

**Discussion by mvenetos** [weather-gov/api#311] [Q&A]
In experimenting with the  gridpoints endpoint of the weather.gov API, I have noticed it doesn't seem to match results I get for the same location at https://forecast.weather.gov/MapClick.php.  Why?  This is the specific location, and URL I am using: 

https://api.weather.gov/gridpoints/GJT/154,103 

The data that is returned has more hours where the temperature is unchanged than I see at forecast.weather.gov.  

I am also curious if the units of the gridpoints call can be specified to get US units instead of SI?  

Finally it looks like the hourly forecast endpoint( https://api.weather.gov/gridpoints/LWX/96,70/forecast/hourly ) might be closer to what I'd like to have but it does not appear to provide the relativeHumidity or skyCover parameters?  Is there a way to add additional layers to an hourly forecast endpoint call?  

Thank you in advance.

**Comment by scadergit** [comment]
The current forecast is a completely separate system that the API. The current forecast parses the netcdf gridded data directly, whereas the API calls a "forecast web service" (or fcws) to get the gridded data. The fcws was written 3 years ago, and is scheduled to be sunset in favor of a newer, simpler solution. We recognize the importance of this effort, but we do not set the priority.

If you [review the API openapi specification](https://api.weather.gov/openapi.json), you'll see the /gridpoint/.../forecast endpoint allows a parameter named "units" with values "us" or "si". 

The /forecast and /forecast/hourly endpoints are scheduled to be enhanced, but they currenly only reproduce what was provided on the current MapClick.php forecast page.
